amoro-core/pom.xml

```diff
@@ -149,6 +149,7 @@
         <dependency>
             <groupId>org.rocksdb</groupId>
             <artifactId>rocksdbjni</artifactId>
+            <scope>${rocksdb-dependency-scope}</scope>
```

Comment:
I'm not sure if we need to emphasize the scope here. Perhaps it can be inherited directly from the parent?

Reply:
this need to be defined here, otherwise the compile will be failed because of can found rocksdb class.